app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    @wraps(func)
+    def wrapper(*args) -> Any:
+        if args in cache_store:
```

Comment:
**kwargs and function name should be included as well

Reply:
I have added **kwargs, what did you mean by "function name"?

Reply:
fixed the code, please check